### ISSUE TITLE
Advanced search form: ARIA attributes, focus order, and tooltips

### ISSUE THREAD

**Issue by aviannamiller** [omeka-s-themes/lively#24]
#20 discusses color contrast issues on the advanced search form. This issue deals with more a11y problems with the advanced search form on Lively.

- Invalid ARIA attribute `aria-title` appears on `numeric-toggle-time button` elements (five total instances). Perhaps an `aria-label` was intended here?
- `select[name="has_media"]` is missing an accessible name. A `label` / `aria-label` is needed here, perhaps also a title.
- There is an illogical focus order after `.add-value` buttons are activated. Focus stays on the trigger element (the button) and progresses to the next element after tabbing. Instead, focus should move to the newly added element and its appropriate field items.
- Tooltips are inaccessible via keyboard / screen reader. See [MDN Web Docs on ARIA: tooltip role
](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Reference/Roles/tooltip_role) for more.

**Comment by aviannamiller** [comment]
Thanks for the changes. Upon deleting a value, I'm seeing focus move to the add value button, regardless of any values in between the removed and the button. It would be best if focus progressed sequentially. If there are no values after the deleted value, then it would be appropriate to move right to the add value button. If there are values in between, it should move directly to the next focusable element, the input(s).

This is for both themes.
